src/main.rs

```diff
@@ -288,55 +304,108 @@ async fn main() {
                 }
             }
 
-            FileCommand::List { file_path, all } => {
+            FileCommand::List { storage, all } => {
+                let root = provide_root(&None);
+                let storage = storage.as_ref().unwrap_or(&root);
+
                 if !all {
```

Comment:
I don't really get it why `-a` flag splits "official" storage names into 2 different groups (`tags`, `stats`, `app_id`, `favorites` vs. `stats`, `properties` etc.). I think, the two groups should be merged and treated as one.